src/legacy/pages/em/classification/sessionclassificationpage.tsx

```diff
@@ -85,6 +85,8 @@ export default function SessionClassificationPage() {
                         .sort((a: Classification, b: Classification) => {
                           return b.classDistribution - a.classDistribution;
                         })
+
```

Comment:
Setting a responsive colum size would avoid having to do this. Something like `<Col key={i} xs={12} sm={6} lg={4} xl={3} xxl={2}>`  seems to render well in all sizes.
Otherwise, setting `<Col key={i} xs={'auto'}>` will make the classes always appear as small as they can be, which is very similar to what happens when adding many empty columns.

Reply:
Note, the `margin: 2` property will mess with bootstrap because it will make the column's element take more space than expected. To increase the spacing, it is better to set a `padding` value (bootstrap already sets one, so there already is spacing by default, but you can make it larger or smaller)